unit_tests/system/python_loader_test.cpp

```diff
@@ -49,23 +49,29 @@ TEST(PythonLoader, print_error) {
         "Message: global name 'a' is not defined\nTraceback");
 }
 
+
+// only test embedded python if we actually copied out python
```

Comment:
Change comment to Test that necessary python libraries are available in embedded python.